docs/tutorials/fundamentals/installation.md

```diff
@@ -42,15 +39,6 @@ You may also want:
 - an environment manager like [conda](https://docs.conda.io/en/latest/miniconda.html) or
 [venv](https://docs.python.org/3/library/venv.html) **(Highly recommended)**
 
-### Prerequisites for installing napari as a bundled app
-This is the easiest way to install napari if you only wish to use it as a standalone GUI app.
-This installation method does not have any prerequisites.
-
-[Click here](#install-as-a-bundled-app) to see instructions
-for installing the bundled app.
-
-## Install as Python package (recommended)
-
 Python package distributions of napari can be installed via `pip`, `conda-forge`, or from source.
```

Comment:
```suggestion
````{note} 
While we recommend installing as a Python package, you can also install napari as [a conda bundled app](#install-as-a-bundled-app).
````

Python package distributions of napari can be installed via `pip`, `conda-forge`, or from source.
```


I agree with the logic, but I'd like to keep the early mention of the bundle so it's a bit more accessible. How's this for a compromise?